feeder/parsers/IS.py

```diff
@@ -0,0 +1,67 @@
+import requests
+from datetime import datetime
+from collections import defaultdict
+import xml.etree.ElementTree as ET
+import json
+
+STATIONS = {
+
+    # http://www.landsvirkjun.com/company/powerstations/
+    'Bjarnaflag': 'geothermal',    # No data?
+    'BLANDA_F': 'hydro',
+    'BUDAR_O': 'hydro',
+    'BURF_F': 'hydro',
+    'FLJOTSDA': 'hydro',
+    'Hafio': 'wind',            # No data?
+    'HRAUN_F': 'hydro',
+    'IRAFOS_F': 'hydro',
+    'KRAFLA_F': 'geothermal',
+    'LAXA_F': 'hydro',
+    'LAXARVAT': 'hydro',
+    'laxa': 'hydro',            # No data?
+    'LJOSIF_F': 'hydro',
+    'SIG_F': 'hydro',
+    'Steingrimsstod': 'hydro',  # No data?
+    'SULTAR_F': 'hydro',
+    'VATNSHAM': 'hydro',
+
+    # https://en.wikipedia.org/wiki/List_of_power_stations_in_Iceland
+    'KOLVID': 'geothermal',
+    'LAGARF': 'hydro',
+    'MJOLKA': 'hydro',
+    'REY': 'geothermal',
+    'X_NESJAV': 'geothermal',
+    'SVA': 'geothermal',
+}
+
+def fetch_production(country_code='IS', session=None):
+
+    # Query Landsnet for latest power production data for Iceland
+    r = session or requests.session()
+    url = 'http://amper.landsnet.is/MapData/api/measurements'
+    response = r.get(url)
+    json_obj = json.loads(response.text)
+
+    production = defaultdict(float)
+
+    # Calculate production values for each power station
+    for key, value in STATIONS.items():
+        items = [item for item in json_obj if item["substation"] == key and item["MW"] >= 0]
```

Comment:
They are ignored, yeah. I'm not sure it's a good idea to funnel them all into unknown by default because I don't think everything we get through that API is actually power generation.

For example, there is a substation `BRENNIME` which Google translates/expands to `Brennimelur` which seems to be a capacitor bank: http://www.rst.is/en/verkefni/2001/BRE-30.html not a generator/power station.

The data for that example has some pretty big MW values, so it'd skew our results quite heavily. Without better knowledge of Icelandic I was thinking it might be best for now to just stick with the stations that @birkir made specific note of? What do you think?